[2021-04-17 06:48:16] local.ERROR: Uncaught ErrorException: Declaration of App\Http\Middleware\Securitycheck::handle(Illuminate\Http\Request $request, Closure $next) should be compatible with Illuminate\Auth\Middleware\Authenticate::handle($request, Closure $next, ...$guards) in C:\xampp\htdocs\feedThePoor\app\Http\Middleware\Securitycheck.php:18
Stack trace:
#0 C:\xampp\htdocs\feedThePoor\vendor\composer\ClassLoader.php(444): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Declaration of ...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\feedThePoor\vendor\composer\ClassLoader.php(444): include()
#2 C:\xampp\htdocs\feedThePoor\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#3 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Http\\Middle...')
#4 [internal function]: spl_autoload_call('App\\Http\\Middle...')
#5 Command line code(1): ReflectionMethod->__construct('App\\Http\\Middle...', 'handle')
#6 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught ErrorException: Declaration of App\\Http\\Middleware\\Securitycheck::handle(Illuminate\\Http\\Request $request, Closure $next) should be compatible with Illuminate\\Auth\\Middleware\\Authenticate::handle($request, Closure $next, ...$guards) in C:\\xampp\\htdocs\\feedThePoor\\app\\Http\\Middleware\\Securitycheck.php:18
Stack trace:
#0 C:\\xampp\\htdocs\\feedThePoor\\vendor\\composer\\ClassLoader.php(444): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError(2, 'Declaration of ...', 'C:\\\\xampp\\\\htdocs...', 18, Array)
#1 C:\\xampp\\htdocs\\feedThePoor\\vendor\\composer\\ClassLoader.php(444): include()
#2 C:\\xampp\\htdocs\\feedThePoor\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#3 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Http\\\\Middle...')
#4 [internal function]: spl_autoload_call('App\\\\Http\\\\Middle...')
#5 Command line code(1): ReflectionMethod->__construct('App\\\\Http\\\\Middle...', 'handle')
#6 {main}
  thrown at C:\\xampp\\htdocs\\feedThePoor\\app\\Http\\Middleware\\Securitycheck.php:18)
[stacktrace]
#0 {main}
"} 
